PKGBUILD

```diff
@@ -0,0 +1,22 @@
+# Maintainer: Your Name <youremail@domain.com>
+pkgname="particle-life"
+pkgrel=1
+pkgver=1.1.3 # todo: make automatic
```

Comment:
The latest version of Particle Life App is currently 1.1.2, but I assume that you wait for 1.1.3 which includes the changes that make it work on linux, right?

Reply:
aye - there is a way to make that into a fn call to always be at the latest release: I have not gotten that to actually work yet.